tikv-configuration-file.md

```diff
@@ -1108,7 +1108,7 @@ Configuration items related to `rocksdb.defaultcf.titan`
 
 ## rocksdb.writecf
 
-Configuration items related to `rocksdb.writecf`
+Configuration items related to `rocksdb.writecf`, which are identical to `rocksdb.defaultcf`. Those not listed here have the same default value and are recommended to be set the same as `rocksdb.defaultcf`.
```

Comment:
"Those not listed here have the same default value and are recommended to be set the same as `rocksdb.defaultcf`."

This may not be always true.

Reply:
There are two parts in this statement. I think at least we should make sure the first part stay true. For master branch I've checked and repaired all the violating items.

Reply:
then how about removing "and are recommended to be set the same as `rocksdb.defaultcf`."?